pkg/app/options.go

```diff
@@ -42,3 +42,12 @@ func WithInitializers(initializers ...workflow.ExtensionInit) Opts {
 		}
 	}
 }
+
+func WithRuntimeInfo(name, version string) Opts {
```

Comment:
I'm not sure about the signature; if the `RuntimeInfo` struct will be extended (which is very likely) this will mean a breaking change to this function. Might be better to pass in `workflow.RuntimeInfo` instead? @PeterSchafer

Reply:
Question/Suggestion: Why not use RuntimeInfo as a parameter instead of the individual values.

Reply:
I think that's what my comment was suggesting, too. Will change to that then.